src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents the exit command to end the program.
+ */
+public class ExitCommand extends Command {
+
+    public ExitCommand() {}
```

Comment:
Might be cleaner to leave it as [default constructor](https://beginnersbook.com/2014/01/default-constructor-java-example/) instead? Only scenario I can imagine to have this line is if we want to keep our constructor `private` so that other classes will not be able to call this constructor.